content/docs/start/experiments.md

```diff
@@ -167,25 +170,25 @@ increase number of features:
 +  ngrams: 2
 ```
 
-And the beauty of the `dvc.yaml` file is that all you need to do now is to run:
+And the beauty of `dvc.yaml` is that all you need to do now is to run:
 
 ```dvc
 $ dvc repro
 ```
 
-It'll analyze changes, will analyze existing cache of previous runs and will run
-only commands that are needed to get the new result (model, metrics, plots).
+It'll analyze the changes, use existing cache of previous runs, and execute only
+the commands that are needed to get the new results (model, metrics, plots).
 
-The same logic applies to other possible experiment adjustments - edit source
-code, edit or update dataset - you do the changes, run `dvc repro` and it runs
-what needs to be run.
+The same logic applies to other possible experiment adjustments — edit source
+code, update datasets — you do the changes, use `dvc repro`, and DVC runs what
+needs to be run.
 
 ## Comparing experiments
 
-Finally, we are now ready to compare experiments. DVC has a few commands to see
-metrics and parameter changes, visualize plots for one or more experiments.
-Let's compare the last "bigrams" run with the last committed "baseline"
-iteration:
+Finally, we are now ready to compare results. DVC has a few commands to see
```

Comment:
we compare everything, not results only

Reply:
Agree. We had "compare experiments" before and I thought "results" is a more general concept. Should we roll back? Or just say "ready to compare everything"?

Reply:
Going for "compare everything!" for now.